crates/gpui/src/platform/windows/window.rs

```diff
@@ -0,0 +1,557 @@
+        unsafe { SetWindowTextW(self.inner.hwnd, &HSTRING::from(title)) }
+            .inspect_err(|e| log::error!("Set title failed: {e}"))
+            .ok();
+    }
+
+    // todo!("windows")
+    fn set_edited(&mut self, edited: bool) {}
+
+    // todo!("windows")
+    fn show_character_palette(&self) {}
+
+    // todo!("windows")
+    fn minimize(&self) {}
+
+    // todo!("windows")
+    fn zoom(&self) {}
+
+    // todo!("windows")
+    fn toggle_full_screen(&self) {}
+
+    // todo!("windows")
+    fn on_request_frame(&self, callback: Box<dyn FnMut()>) {
+        self.inner.callbacks.borrow_mut().request_frame = callback.into();
+    }
+
+    // todo!("windows")
+    fn on_input(&self, callback: Box<dyn FnMut(PlatformInput) -> bool>) {
+        self.inner.callbacks.borrow_mut().input = callback.into();
+    }
+
+    // todo!("windows")
+    fn on_active_status_change(&self, callback: Box<dyn FnMut(bool)>) {
+        self.inner.callbacks.borrow_mut().active_status_change = callback.into();
+    }
+
+    // todo!("windows")
+    fn on_resize(&self, callback: Box<dyn FnMut(Size<Pixels>, f32)>) {
+        self.inner.callbacks.borrow_mut().resize = callback.into();
+    }
+
+    // todo!("windows")
+    fn on_fullscreen(&self, callback: Box<dyn FnMut(bool)>) {
+        self.inner.callbacks.borrow_mut().fullscreen = callback.into();
+    }
+
+    // todo!("windows")
+    fn on_moved(&self, callback: Box<dyn FnMut()>) {
+        self.inner.callbacks.borrow_mut().moved = callback.into();
+    }
+
+    // todo!("windows")
+    fn on_should_close(&self, callback: Box<dyn FnMut() -> bool>) {
+        self.inner.callbacks.borrow_mut().should_close = callback.into();
+    }
+
+    // todo!("windows")
+    fn on_close(&self, callback: Box<dyn FnOnce()>) {
+        self.inner.callbacks.borrow_mut().close = callback.into();
+    }
+
+    // todo!("windows")
+    fn on_appearance_changed(&self, callback: Box<dyn FnMut()>) {
+        self.inner.callbacks.borrow_mut().appearance_changed = callback.into();
+    }
+
+    // todo!("windows")
+    fn is_topmost_for_position(&self, position: Point<Pixels>) -> bool {
+        true
+    }
+
+    // todo!("windows")
+    fn draw(&self, scene: &Scene) {
+        self.inner.renderer.borrow_mut().draw(scene)
+    }
+
+    // todo!("windows")
+    fn sprite_atlas(&self) -> Arc<dyn PlatformAtlas> {
+        self.inner.renderer.borrow().sprite_atlas().clone()
+    }
+
+    // todo!("windows")
+    fn set_graphics_profiler_enabled(&self, enabled: bool) {}
+}
+
+fn register_wnd_class() -> PCWSTR {
+    const CLASS_NAME: PCWSTR = w!("Zed::Window");
+
+    static ONCE: Once = Once::new();
+    ONCE.call_once(|| {
+        let wc = WNDCLASSW {
+            lpfnWndProc: Some(wnd_proc_start),
+            cbClsExtra: 0,
+            cbWndExtra: std::mem::size_of::<*const c_void>() as i32,
+            hInstance: HINSTANCE::default(),
+            hIcon: HICON::default(),
+            hCursor: HCURSOR::default(),
```

Comment:
This will show the arrow cursor when hovering over the window, rather than the current confusing loading cursor. Though we will need to pass `None` here later when we implement updating the cursor icon on-demand.

```suggestion
            hCursor: unsafe { LoadCursorW(None, IDC_ARROW).ok().unwrap() },
```

Reply:
I did it